pawno/include/modules/src/textdraws.inc

```diff
@@ -1387,16 +1409,22 @@ public DoShowIntroTextDraws(index, response_code, data[])
 
 ShowIntroTextDraws(playerid)
 {
+    #pragma unused playerid
+    /*
 	HTTP(playerid, HTTP_GET, "infinite-gaming.ml/khk/bulletproof/league/news_textdraw_text.php", "", "DoShowIntroTextDraws");
 
 	TextDrawShowForPlayer(playerid, Intro_GoWebsite);
 	TextDrawShowForPlayer(playerid, Intro_GetStarted);
 	TextDrawShowForPlayer(playerid, Intro_News);
+    */
```

Comment:
I don't think league should be enabled by default as it's not very popular. And people congratulated me for removing league system in my server.